plenum/server/node.py

```diff
@@ -2284,6 +2283,8 @@ def allLedgersCaughtUp(self):
         # `self.num_txns_caught_up_in_last_catchup()`
         self.processStashedOrderedReqs()
         self.mode = Mode.synced
+        for replica in self.replicas.values():
```

Comment:
Should we move it to `caught_up_till_3pc` and `catchup_clear_for_backup`?
BTW maybe we should combine them so that
`caught_up_till_3pc` calls
```
if not replica.isMaster:
    self._catchup_clear_for_backup()
    return
<...>
```